tests/test_data/rewe.de/rewe.json

```diff
@@ -0,0 +1,39 @@
+{
+  "author": null,
```

Comment:
Hmm; some kind of bug in our `SchemaOrg.author` method, possibly.  I think an author name should be appearing here.

Reply:
The rewe.de/rezepte recipes don't have an author

Reply:
Ok, yep.  The author is listed in the `WebPage` metadata.. but for some reason we're not retrieving it from there.  It's not a bug in your code; I'm going to spend some time investigating that.

Reply:
@Gamekohl could you try merging the latest changes from the `main` branch?  With those in place, I believe the `author` value should now appear.

Reply:
@jayaddison I merged and adjusted it accordingly